isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -436,15 +455,19 @@ namespace Isis {
     // make sure there is a node for every measure in this measure's parent
```

Comment:
I know this is existing functionality, but I'm confused by we have to check this here. If you're adding a whole new point, then it will use pointAdded. So, this is only adding a single new measure/image.

I think we can get rid of this check entirely.

Reply:
:+1: Just to confirm, I did take this out.